ofborg/src/easylapin.rs

```diff
@@ -117,6 +115,20 @@ impl<'a, W: SimpleWorker + 'a> ConsumerExt<'a, W> for CloseOnDrop<Channel> {
     }
 }
 
+// Same as a regular channel, but without prefetching,
+// used for services with multiple instances.
```

Comment:
Might as well make this a doc comment.

```suggestion
/// Same as a regular channel, but without prefetching,
/// used for services with multiple instances.
```